cmd/fossa/main.go

```diff
@@ -39,6 +39,7 @@ var App = cli.App{
 		report.Cmd,
 		test.Cmd,
 		update.Cmd,
+		analyze.NewCmd,
```

Comment:
I think we should comment this out in the merged code. I don't know if we should be displaying another command when `fossa -h` is run.